indexstyling.css

```diff
@@ -13,20 +13,92 @@ html {
 }
 
 body {
-    background-color: #FAEBD7FF;
+    background-color: #122c34;
     font-size: 2rem;
     font-family: pixel;
+    color: #ffffff;
+}
+
+header {
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+    height: 100px;
+    background-color: #2a4494;
+    border-style: dashed;
+    border-color: #FFFFFFFF;
+    border-width: 5px;
+    border-left-style: none;
+    border-right-style: none;
+}
+
+.title {
+    margin: 10px;
+    display: flex;
+    justify-content: center;
+    font-size: 7rem;
+}
+
+@media only screen and (max-width: 1000px) {
```

Comment:
for consistency, I'd move all of your media queries to the bottom below your main styling